base_external/post-image.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
@shykn1 one suggestion here, when you start a script which heavily leverages another script it's nice to place this in two different commits, one which commits the unmodified script and a second which adds your changes.  The benefit of doing this way is that as we update to use new versions of buildroot it will be easy to see which part of the script needs to be changed by looking at the commit history, and may be possible to simply apply a single patch to re-apply the changes.